vite.config.js

```diff
@@ -0,0 +1,13 @@
+import { defineConfig } from "vite";
+import eslint from 'vite-plugin-eslint';
+import react from '@vitejs/plugin-react';
+
+
+
+export default defineConfig({
```

Comment:
Can you sync up with IDE's config https://github.com/NucleoidJS/IDE/blob/f126022b81171c0fac20c3451b90d715017f658b/vite.config.js?